server/middleware/setUpSentry.ts

```diff
@@ -1,15 +1,23 @@
 import * as Sentry from '@sentry/node'
-import '@sentry/tracing'
+import * as Tracing from '@sentry/tracing'
 import express from 'express'
 import applicationVersion from '../applicationVersion'
 import config from '../config'
 
 export function setUpSentryRequestHandler(app: express.Express): void {
   if (config.sentry.dsn) {
+    const router = express.Router()
```

Comment:
I think you'll need to call  `app.use(router)` for this to do anything. In make-recalls-ui, setupSentry returns the router, and `app.use(setUpSentry())` is called in app.ts

Reply:
Also, I expect the call to `router.use(Sentry.Handlers.tracingHandler())` in make-recalls-ui is significant

Reply:
Ah I didn't see this feedback before pushing my latest changes. I did end up removing this line 👍 

And yes that line was very significant as it turns out 😆